src/Resources/public/sonata-page.back.js

```diff
@@ -986,4 +1005,4 @@
         $(defaultRegistry)[pluginName]();
     });
 
-})( jQuery, window, document );
+})( jQuery, window, document );
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.

Reply:
Please address this

Reply:
Woops I merged with this change… oh well…